Source/UserManagement/Web/Controllers/UserController.cs

```diff
@@ -14,7 +14,7 @@
 
 namespace Web
 {
-    [Route("api/usermanagement/")]
+    [Route("api/staffuser")]
```

Comment:
Restfull api naming is typically plural, so staffusers is more the norm :)

Reply:
Okay, I will keep that in mind then.